source/iostreams.tex

```diff
@@ -12215,7 +12215,8 @@
 \begin{itemdescr}
 \pnum
 \effects
-Calls \tcode{sb.emit()}.
+Behaves as an unformatted output function\iref{ostream.unformatted}.
+After constructing a sentry object, calls \tcode{sb.emit()}.
```

Comment:
The issue has `sentry` in code font but it looks like "After constructing a sentry object" is commonly in non-code font in the existing text. The slightly more general "sentry object" has both. Should probably be a separate editorial issue.